layouts/partials/head.html

```diff
@@ -1,6 +1,6 @@
 <head>
   <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width initial-scale=1" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
This checks out: https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag#viewport_basics